src/test/kotlin/it/unibo/protelis/web/execution/ProtelisNodeTest.kt

```diff
@@ -0,0 +1,49 @@
+package it.unibo.protelis.web.execution
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.vertx.core.json.Json
+import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
+import kotlin.test.assertNotNull
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+class ProtelisNodeTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [ProtelisNodeTest] at /src/src/test/kotlin/it/unibo/protelis/web/execution/ProtelisNodeTest.kt:11:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)